scipy/stats/_continuous_distns.py

```diff
@@ -6590,12 +6593,20 @@ def _stats(self, nu):
         return mu, mu2, g1, g2
 
     def _entropy(self, nu):
-        A = sc.gammaln(nu) - np.log(2)
-        B = -0.5 * np.log(nu)
-        C = (2 * nu - (2 * nu - 1) * sc.digamma(nu)) / 2
+        shape = np.shape(nu)
+        # because somehow this isn't taken care of by the infrastructure...
+        nu = np.atleast_1d(nu)
+        A = sc.gammaln(nu)
+        B = nu - (nu - 0.5) * sc.digamma(nu)
+        C = -0.5 * np.log(nu) - np.log(2)
         h = A + B + C
-
-        return h
+        # This is the asymptotic sum of A and B (see gh-17868)
+        norm_entropy = stats.norm._entropy()
+        # Above, this is lost to rounding error for large nu, so use the
+        # asymptotic sum when the approximation becomes accurate
+        i = nu > 1e7  # roundoff error ~ approximation error
+        h[i] = C[i] + norm_entropy
```

Comment:
@steppi is this what you had in mind?
```suggestion
        h[i] = C[i] + norm_entropy - 1/(12*nu[i])
```
And you see that there is a cutoff for using the expansion of `1e7`, but you mean to tune it again based on the addition of this new term? ISTM that the cutoff would probably be lower now because the approximation is more accurate, right?

Reply:
yeah, that looks right, and yes for tuning the cutoff again